app/src/main/resources/index.html

```diff
@@ -6,12 +6,19 @@
   </head>
   <body>
     <div class="buttonTypeSelector">
-      Desktop mode:
-      <label for="buttonTypeRedirect">Redirect (default)</label>
-      <input type="radio" id="buttonTypeRedirect" name="buttonTypeSelector" value="REDIRECT" checked />
-      <label for="buttonTypePopup">Popup</label>
-      <input type="radio" id="buttonTypePopup" name="buttonTypeSelector" value="POPUP" />
-      <small>* Mobile is always redirect</small>
+        Desktop mode:
+        <label for="buttonTypeRedirect">Redirect (default)</label>
+        <input type="radio" id="buttonTypeRedirect" name="buttonTypeSelector" value="REDIRECT" checked />
+        <label for="buttonTypePopup">Popup</label>
+        <input type="radio" id="buttonTypePopup" name="buttonTypeSelector" value="POPUP" />
+        <small>* Mobile is always redirect</small>
+    </div>
+    <div class="transferTypeSelector">
```

Comment:
maybe add some margin at the bottom of `.transferTypeSelector` in stylesheet to separate it from the button

Reply:
Actually I was wondering about that, how would you do that? Should I add a new line or something like that?

Reply:
Yes it's similar to the styles from `. buttonTypeSelector`:
```
. transferTypeSelector {
    display: block;
    margin-bottom: 20px;
}

.transferTypeSelector label {
    cursor: pointer;
    margin-left: 10px;
}
```

Reply:
thank you, CSS has been mysterious to me :)